.github/workflows/ci.yaml

```diff
@@ -32,6 +32,20 @@ jobs:
       - run: npm run get-data
       - run: npm run get-narratives
       - run: npm run integration-test:ci
+  smoke-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [10]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - run: npm install
```

Comment:
@bcoe Any reason why we shouldn't use `npm ci` here?

Reply:
@eharkins looks like `npm ci` works like a charm 👍